catt/controllers.py

```diff
@@ -26,6 +26,20 @@ def get_stream_info(video_url):
     return stream_info
 
 
+def get_chromecast(device_name):
+    devices = pychromecast.get_chromecasts()
+    if device_name:
```

Comment:
Sorting devices is always needed, because you don't want to rely on the network-returned order, even when you're specifying a device name, because we might want to do an `in` later on. This is fine for now, though, I can change it when/if I implement the `in`.